swagger/src/main/scala/org/http4s/rho/swagger/SwaggerSupport.scala

```diff
@@ -17,10 +17,10 @@ object SwaggerSupport {
    * representing the full API
    */
   def apply(
-    swaggerFormats: SwaggerFormats = DefaultSwaggerFormats,
-    apiPath: TypedPath[HNil] = "swagger.json",
-    apiInfo: Info = Info(title = "My API", version = "1.0.0"),
-    swaggerRoutesInSwagger: Boolean = false): RhoMiddleware = { routes =>
+             swaggerFormats: SwaggerFormats = SwaggerFormats.defaultSwaggerFormats,
```

Comment:
Can we undo the indent change? I'm open to being convinced this is a good idea but haven't seen this style.